tests/ops/qubit/test_parametric_ops.py

```diff
@@ -927,6 +951,57 @@ def test_rz(self, tol):
         assert np.allclose(qml.RZ.compute_matrix(np.pi), -1j * Z, atol=tol, rtol=0)
         assert np.allclose(qml.RZ(np.pi, wires=0).matrix(), -1j * Z, atol=tol, rtol=0)
 
+    @pytest.mark.parametrize("dim", range(3))
+    @pytest.mark.parametrize("wires", (range(2), range(3)))
+    @pytest.mark.parametrize("phi", np.linspace(-np.pi, np.pi, 10))
+    def test_pcphase(self, phi, dim, wires):
+        """Test that the PCPhase operator matrix looks correct."""
+        num_wires = len(wires)
+        op = qml.PCPhase(phi, dim=dim, wires=wires)
+
+        mat1 = qml.matrix(op)
+        mat2 = op.compute_matrix(*op.parameters, **op.hyperparameters)
+
+        expected_mat = np.diag(
+            [np.exp(1j * phi) if i < dim else 1.0 for i in range(2**num_wires)]
+        )
+        assert np.allclose(mat1, expected_mat)
+        assert np.allclose(mat2, expected_mat)
+
+    # @pytest.mark.tf
```

Comment:
Are these tests expected to be added in this PR?

Reply:
Added 👍🏼